lib/game.rb

```diff
@@ -0,0 +1,131 @@
+require './lib/card'
+require './lib/deck'
+require './lib/player'
+require './lib/turn'
+
+class Game
+    def initialize    
+    end
+
+    def welcome_message 
+        puts "Welcome to War! (or Peace) This game will be played with 52 cards."
+        puts "The players today are Megan and Aurora."
+        puts "Type 'GO' to start the game!"
+        puts "------------------------------------------------------------------"
+        user_input = gets.chomp.upcase
+
+        if user_input == "GO"
```

Comment:
You have a, "you only get one re-do"-type game lol. You either enter the correct input on the second attempt or you don't get to play. That's fine, but I found that interesting because more often than not, you see users being given multiple attempts to enter incorrect input.